pages/en/lb4/Getting-started.md

```diff
@@ -9,298 +9,82 @@ summary: Write and run a LoopBack 4 "Hello World" project in JavaScript and Type
-
-Install the `@types/node` package as a dependency by entering this command:
-
-```js
-npm i -s @types/node
+cd getting-strted
+npm start
 ```
 
-This adds the `@types/node` package to the `dependencies` property of `package.json`.
+In a browser, visit `http://127.0.0.1:3000/ping`.
 
-### Modify `index.ts`
+## Adding your own controller
 
-Copy the following code into the `index.ts` file, replacing the entire
-contents of the file:
+Now that we have a basic project created, it's time to add our own [controller](Controllers.html).
+Let's add a simple "Hello World" controller as follows:
 
-{% include code-caption.html content="index.ts" %}
-```ts
-import {Application} from '@loopback/core';
-import {RestComponent, RestServer} from '@loopback/rest';
+* Create a new file in `/src/controllers/` called `hello.controller.ts`.
 
-const app = new Application({
-  components: [RestComponent],
-});
-
-(async function start() {
-  // Grab the REST server instance
-  const server = await app.getServer(RestServer);
-  // Setup our handler!
-  server.handler((sequence, request, response) => {
-    sequence.send(response, 'Hello world!');
-  });
-  await app.start();
-  console.log(`REST server listening on port ${server.getSync('rest.port')}`);
-})();
-```
-### Modify `tsconfig.json`
+* Paste the following contents into the file:
+  ```ts
+  import {get} from '@loopback/rest';
 
-Edit `tsconfig.json` as follows:
-
-{% include code-caption.html content="tsconfig.json" %}
-```
-{
-  "compilerOptions": {
-    "target": "es2017", /* For Node.js v8, use "es2017"; for Node.js v6, use "es6" */
-    "module": "commonjs",                     
-    "experimentalDecorators": true,
-    "emitDecoratorMetadata": true
+  export class HelloController {
+    @get('/hello')
+    hello(): string {
+      return 'Hello world!';
+    }
   }
-}
-```
-
-### Build and run the project
-
-Build the project by entering this command:
-
-```
-$ ./node_modules/typescript/bin/tsc
-```
-
-This command compiles the TypeScript in `index.ts` to JavaScript in `index.js`.
-
-If you [added the build command to `package.json`](#optional-edit-packagejson), you can use the command `npm run-script build`.
-
-Run the project by entering this command:
-
-```
-$ node index.js
-```
-
-If you [added the run command to `package.json`](#optional-edit-packagejson), you can use the command `npm run-script start`.
-
-In another console window, enter this command:
-
-```
-$ curl http://localhost:3000/helloworld
-```
-
-Or, load [http://localhost:3000/helloworld](http://localhost:3000/helloworld) in your web browser.
-
-You should see the `Hello world!` message.
-
-Press **Ctrl-C** to stop the application.
+  ```
+
+* Update `/src/application.ts` to load the controller:
+    * Import `HelloController` at the top of the file
+      ```ts
+      import {HelloController} from '.controllers/hello.controller';
+      ```
+
+    * Bind controller in `setupControllers()`
+      ```ts
+      setupControllers() {
```

Comment:
Correct me if I'm wrong, but we should specify that this method needs to be called in order to setup the binding right?

Reply:
Changed it to say **Add** instead of **Bind**

The function is already called by the default scaffolding set up by the project. They just need to add the one line in it but I'm showing the whole function for context.